rotkehlchen/tests/unit/accounting/test_misc.py

```diff
@@ -180,3 +185,42 @@ def test_fees_in_received_asset(accountant, google_service):
         AccountingEventType.LEDGER_ACTION: PNL(taxable=FVal('178.615'), free=ZERO),
     })
     check_pnls_and_csv(accountant, expected_pnls, google_service)
+
+
+@pytest.mark.parametrize('should_mock_price_queries', [False])
+def test_native_currency_is_respected(accountant: 'Accountant') -> None:
```

Comment:
What is native currency? Do you mean profit_currency/main_currency? Do not introduce new terms. Use the term we use in the rest of the code